tests/comments_closure_typecast/closure-compiler-type-cast.js

```diff
@@ -69,3 +69,6 @@ const style2 =/**
 const v6 = /**@type {string} */(value);
 const v7 = /** @type{string} */(value);
 const v8 = /**@type{string} */(value);
+
+const v9 = /** @type {string} */
+  (value);
```

Comment:
Maybe for `@type` comments (for all JSDoc comments)  better to keep it in one line

/cc @thorn0

Reply:
Yeah, why not do this for all comments?

Reply:
Looks good. I'll fix.
```js
// Input
const foo = /* foo */
  bar;

// Expected output😄
const foo = /* foo */ bar;
```

Reply:
done!